Makefile

```diff
@@ -53,6 +53,13 @@ lint: ## golangci-lint
 	$(call print-target)
 	golangci-lint run --fix
 
+.PHONY: check
+check: ## govulncheck
+	$(call print-target)
+	govulncheck -scan module
+	govulncheck -scan package ./...
+	govulncheck ./...
```

Comment:
Isn't the symbol level the most detailed and the only one that is actually needed?

```suggestion
	govulncheck ./...
```

Reply:
From testing it (e.g., with an older version of go), I did not get the same output with the three scans.
Feel free to remove it if you think they are redundant 😄

Reply:
AFAIK `symbol` is the most detailed  and gets rid of possible false-positives. It should not report issues if the vulnerable code is not on the execution path. I believe that other levels only analyze the "imports" or module dependencies.

Removing. Please let me know if you think I am wrong.